src/adapters/mod.rs

```diff
@@ -5,6 +5,9 @@
 /// An adapter providing time services.
 pub mod clock;
 
+/// An adapter displaying messages on the console.
+pub mod console;
```

Comment:
Since this looks like a debugging feature, maybe it would be good to enable this only when `CARGO_LOG=debug` or to add a --debug cmd line option.